config/nvidia-flags

```diff
@@ -106,7 +107,8 @@ if test "X-nvc" = "X-$cc_vendor" -o "X-nvcc" = "X-$cc_vendor"; then
     ################
 
     HIGH_OPT_CFLAGS="-O1"         # -O2+ currently has test failures.
```

Comment:
Looks like we already did this previously for Autotools builds, which is why the NVHPC action passes tests for Autotools builds, but the tests were disabled for CMake builds